client-ts/signalr-protocol-msgpack/rollup.config.js

```diff
@@ -4,5 +4,6 @@
 import baseConfig from "../rollup-base"
 
 export default baseConfig(__dirname, {
+    ["@aspnet/signalr"]: "signalR",
```

Comment:
?

Reply:
Cleaning up `rollup` grumblings that weren't blocking the build. I'll revert and put in a separate PR.